bigbone-rx/src/main/kotlin/social/bigbone/rx/RxPushNotificationMethods.kt

```diff
@@ -0,0 +1,87 @@
+package social.bigbone.rx
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.api.entity.WebPushSubscription
+import social.bigbone.api.method.PushNotificationMethods
+
+/**
+ * Reactive implementation of [PushNotificationMethods].
+ * Allows access to API methods with endpoints having an "api/vX/push" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/push/">Mastodon push notification API methods</a>
+ */
+class RxPushNotificationMethods(client: MastodonClient) {
+
+    private val pushNotificationMethods = PushNotificationMethods(client)
+
+    @JvmOverloads
```

Comment:
Hello @G10xy Would be great if you could add KDoc for the methods in this class. Please have a look how it is done in other Rx classes (I know they do not all have KDoc yet, but this issue will be addressed in the future). Thx :)